opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-common.ts

```diff
@@ -0,0 +1,14 @@
+import { TaxiiAuthType } from '../../generated/graphql';
+import { FunctionalError } from '../../config/errors';
+
+export const verifyIngestionAuthenticationContent = (authenticationType: string, authenticationValue: string) => {
+  if (authenticationType && authenticationValue) {
+    if (authenticationType === TaxiiAuthType.Basic && authenticationValue.split(':').length !== 2) {
```

Comment:
this is checking for TaxiiAuthType, but since this method is also now used by csv's, we should add CsvAuthType.Basic also i think? even if both enums have the same value right now